lib/generator.js

```diff
@@ -637,7 +638,10 @@ class Generator {
         if (!valid) return log.debug(`${relativeSourceFile} was not generated because condition specified for this file in template configuration in conditionalFiles matched`);
       }
     }
-    if (this.isNonRenderableFile(relativeSourceFile)) return await copyFile(sourceFile, targetFile);
```

Comment:
is it because of prettier?

Reply:
I don't like the if-s without brackets 😄

Reply:
but this is a one-liner? Do you have issues with all ifs? even one liner? 😮

Reply:
for the record, I love them 😄 I'm ok to accept whatever some tool enforces though

Reply:
I can change it to inline 😄

Reply:
Changed hah 😄